src/omero_cli_render.py

```diff
@@ -134,6 +134,51 @@
     # be updated individually.
 """
 
+BATCHSET_HELP = """Set rendering settings from a hierachy of rendering
+    definitions corresponding to an OMERO container hierarchy.
+
+    'projects' and 'datasets' are supported as top level containers, 'name'
+    must be provided.
+    If multiple containers have the same name they will all be processed.
+    Top level keys beginning with '_' are ignored, so they could for example
+    be used as YAML anchors.
+
+    Definitions can be applied to projects, datasets, or datasets in projects.
+
+    'renderdef' is a rendering definition as passed to 'set', except that
```

Comment:
I think keepin the renderdef self-contained will make this spec easier to update, otherwise when adding a key to either the container definition or the renderdef spec you'll need to consider future conflicts or potential confusion.

In addition keeping the renderdef under it's own key means this could potentially be extended to handle other metadata e.g. for setting tags or other annotations.